src/main/java/com/rcv/Tabulator.java

```diff
@@ -234,58 +234,70 @@ private void logSummaryInfo() {
   // logic only considers continuing candidates, so it won't assign any votes to past winners -- but
   // in reality they continue to hold their winning margins for the rest of the rounds, so we need
   // to fill in those values here.
```

Comment:
Just to clarify: "in reality they continue to hold _and potentially increase_ their winning margins"?

Reply:
No, just hold. They can't increase their totals.

Not sure if this is part or all of what you had in mind, but this logic shouldn't apply to the single-winner, keep-going-until-only-two-candidates-remain scenario. In that special case, yes, a winner can keep accumulating votes. I'll update the code to be explicit about only calling `updatePastWinnerTallies` when it's a multi-seat contest.

Reply:
That's helpful, thanks.  I think I'm going to just edit the updatePastWinnerTallies main comment if you don't mind.  Having stepped through in the debugger with the portland_mayor_multi race it makes a lot more sense now.

Reply:
Sure. Do you mean that you want to update this PR yourself and then merge it? Or should I merge it?

Reply:
Actually, I take it back.  On re-re-reading, isn't the end result of calling updatePastWinnerTallies that: previous winners have their round tallies added to the roundTallies object at the winning threshold?  If so, can't we just add a roundTally for past winners of the winningThreshold and skip all the cvr processing?  If I'm wrong, let's discuss in person.  I think you're fine to merge either way, seems to work fine.

Reply:
Ok cool, that makes sense.

Reply:
Do you think this could be more clearly calculated as the winner's surplus is transferred?

Reply:
Good question. I also considered that possibility, and yeah, I think it might be a bit clearer, but it's still a little awkward because we'd have to stash the value somewhere (since it would be happening one round before we were ready to put it in roundTallies) and then look it up later. If I hadn't already gotten the logic in this PR working, though, I would probably do what you're suggesting instead. I can create another low-pri task for this.